cmd/iobench.go

```diff
@@ -135,24 +136,37 @@ func drawHeading(heading string) {
 }
 
 func drawBar(name string, took, ref time.Duration, inputSize uint64, throughput float64) {
+	w, _, err := terminal.GetSize(1)
+	if err != nil {
+		w = 100
+	}
+
 	perc := float64(ref) / float64(took)
-	const cells = 60
 
-	fmt.Printf("%-50s [", name)
+	// take a guess on how big the bar may be.
+	var cells int
+	if barw := w - 40 - 40; barw < cells {
```

Comment:
This heuristics fails on my terminal which 101 symbols wide. I would still argue to put bar plot at the end,
since we have good control over throughput and algorithm fields.

Reply:
I removed the bar completely now since it did not produce so much value by itself. The heuristic you commented on was wrong by the way - it was fixed in the commit after.